p2p_wallet/Scenes/Main/Swap/SwapSettings/ViewModel/SwapSettingsViewModel.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SwapSettingsViewModel.swift
+//  p2p_wallet
+//
+//  Created by Ivan on 28.02.2023.
+//
+
+import Combine
+
+private extension Double {
+    static let minSlippage: Double = 0.01
+    static let maximumSlippage: Double = 50
```

Comment:
minSlippage
maxSlippage

Reply:
we have global const maxSlippage yet, but this have another meaning, not working for this case